Khan/perseus
# View a locked point with mafs

## Summary:
The ability to view a locked points was added as part of
[LEMS-1702](https://khanacademy.atlassian.net/browse/LEMS-1702), but now that we've
switched to mafs, we need to implement viewing a point again with mafs.

- Add a layer for locked figures to the mafs graph
- Createa a new component for said locked feature
- Add an interactive graph story to confirm that locked points can be viewed

Issue: https://khanacademy.atlassian.net/browse/LEMS-1702

## Test plan:
- Open http://localhost:6006/?path=/story/perseus-widgets-interactive-graph--segment-with-mafs-and-locked-points
- Confirm that red points are there at (-7, -7) and (2, -5)
- Drag the points of the line segment over the locked points and confirm that
  the blue points sit on top of the red points. (Check the interactive layer
  is on top of the locked layer.)

[LEMS-1702]: https://khanacademy.atlassian.net/browse/LEMS-1702?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### Story

<img width="917" alt="Screenshot 2024-03-12 at 3 17 13 PM" src="https://github.com/Khan/perseus/assets/13231763/3045a9fe-8b17-4190-b73c-68d72159ae8e">
